{"name": "test_someting2", "status": "failed", "statusDetails": {"message": "AssertionError: The sataus code is wrong", "trace": "self = <tests.test.crud.test_crud.TestCrud object at 0x0000019FDD74C6A0>\nget_booking_id = 4081\n\n    def test_someting2(self,get_booking_id): ##Time pass ignore it\n        url=APIConstants.url_get_search(booking_id=get_booking_id)\n        print(url)\n        print(get_booking_id)\n        response=get_request(url=url,auth=None,in_json=False)\n        print(response.json())\n        actual_status_code=response.status_code\n>       verify_http_status_code(response_data=actual_status_code,expected_data=404)\n\ntest_crud.py:84: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nresponse_data = 200, expected_data = 404\n\n    def verify_http_status_code(response_data, expected_data):\n>       assert response_data == expected_data ,\"The sataus code is wrong\"\nE       AssertionError: The sataus code is wrong\n\n..\\..\\..\\src\\helpers\\common_verification.py:2: AssertionError"}, "attachments": [{"name": "stdout", "source": "a4db387a-3e80-4c2a-949e-f2e0d6b245f1-attachment.txt", "type": "text/plain"}], "start": 1730195931100, "stop": 1730195932171, "uuid": "95eaedfd-f23e-4a03-9cbc-55d724e51500", "historyId": "f96621bee909390f9e3b4e6550927874", "testCaseId": "78c38a059a447b210768886c0f3862a4", "fullName": "tests.test.crud.test_crud.TestCrud#test_someting2", "labels": [{"name": "parentSuite", "value": "tests.test.crud"}, {"name": "suite", "value": "test_crud"}, {"name": "subSuite", "value": "TestCrud"}, {"name": "host", "value": "LAPTOP-J82A4UMN"}, {"name": "thread", "value": "17120-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test.crud.test_crud"}]}